scripts/build-ios.sh

```diff
@@ -88,11 +88,11 @@ mkdir -p build
 pushd build
 
 # Configure CMake project
-SDKROOT="${SDK_ROOT_OVERRIDE:-/Applications/Xcode_13.1.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/}" cmake "$PROJECT_ROOT" -GXcode \
+SDKROOT="${SDK_ROOT_OVERRIDE:-/Applications/Xcode_14.2.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/}" cmake "$PROJECT_ROOT" -GXcode \
```

Comment:
Would it make sense to make a separate PR for this? In case we need to revert.

Reply:
This needs to be reverted.  Result of too much multi-tasking